src/components/pages/Home/UseCasesSection/index.tsx

```diff
@@ -57,9 +51,9 @@ const UseCasesSection: React.ForwardRefRenderFunction<HTMLElement> = () => (
 
         The simplest case of using CML, and a clear way for any user to get started, is to generate a simple report. Add the following .yaml to your project repository and commit to get started
 
-    </Box>
+      </Box>
 
-      <Switch sx={{ my: 4, maxWidth: "160px", mx: "auto" }} />
+      <Switch sx={{ my: 4, maxWidth: "360px", mx: "auto" }} />
```

Comment:
any reason for the size change?

Reply:
To allow the third button. However 240px should suffice

Reply:
Nah it's the right size.

FYI with 240px

<img width="320" alt="image" src="https://user-images.githubusercontent.com/414967/200363351-0f079087-0e90-49a4-8e7e-8a575db5060b.png">